third_party/move/tools/move-package/tests/test_sources/compilation/basic_no_deps/Move.exp

```diff
@@ -37,6 +37,7 @@ CompiledPackageInfo {
             ),
             language_version: None,
             experiments: [],
+            move_2: false,
```

Comment:
I wasn't sure whether to include the flag in third_party.  At first I had trouble getting things to compile without that, but while testing the `aptos` CLI handling I realized it simplified things to leave it out, so I was already doing that.

I did still add handling for this flag (and language/compiler/bytecode-version) in aptos governance, not just third_party, since we will need it eventually, but leave the defaults.

It took me a bit of trying to get Clap's `default_value_if` feature working, and in the process I cleaned up a few other clap lines.  (Oddly, docs I found didn't have underscores/hyphens in the originating parameter, so it took some experimentation to find I needed to refer to it as `"move_2"`, not `"move-2"`.)

Reply:
third_party currently does have any UI. The only UI really is the CLI. 

If we want to have a standard way to access defaults programmatically, this exists already programmatically: https://github.com/aptos-labs/aptos-core/blob/90ee5034322553468ba505f55e8aa7cfa4e39077/aptos-move/framework/src/built_package.rs#L132-L138. Not sure whether code like governance can access just that. The many options passed around are already a knightmare, I would try to minimize.

It is, however, not that super important, and if you want to move forward with the current approach, resolve and re-request review.

Reply:
third_party may not have a UI, but it has some use of `clap` which made me think there was a CLI somewhere.

The challenge is providing the CLI with the controls.  

I've added more use of defaults as you suggest.